src/Build/BackEnd/Components/SdkResolution/SdkResolverContext.cs

```diff
@@ -3,25 +3,26 @@
 
 using System;
 
-using SdkResolverContextBase = Microsoft.Build.Framework.SdkResolverContext;
+using SdkResolverContextBase2 = Microsoft.Build.Framework.SdkResolverContext2;
 
 #nullable disable
 
 namespace Microsoft.Build.BackEnd.SdkResolution
 {
     /// <summary>
-    /// An internal implementation of <see cref="Framework.SdkResolverContext"/>.
+    /// An internal implementation of <see cref="Framework.SdkResolverContext2"/>.
     /// </summary>
-    internal sealed class SdkResolverContext : SdkResolverContextBase
+    internal sealed class SdkResolverContext : SdkResolverContextBase2
```

Comment:
```suggestion
    internal sealed class SdkResolverContext : SdkResolverContextBase
```

OR

```suggestion
    internal sealed class SdkResolverContext : SdkResolverContext2
```

see previous comment. I personally prefer first one, since it makes clear that the imported class is base class.